index.html

```diff
@@ -0,0 +1,219 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet">
+    <link rel="icon" type="image/*" href="assets/images/portfolio.png">
+    <link rel="stylesheet" href="./assets/style/main.css">
+    <title>Mobile version Portfolio</title>
+</head>
+<body>
+    <main class="grid-container">
+        <section class="grid-item grid-item-1 header-section">
+            <nav id="header-nav">
+<span class="nav-logo"><a>My Logo</a></span>
+<span class="nav-toggle"><a>&#9776;</a></span>
+    </nav>
+        </section>
+        <section class="grid-item grid-item-2 introduction-section">
+            <span class="introduction-container">
+            <h1 class="intro-name">I’m Osman<br> Glad to see<br> you!</h1>
+        <p class="intro-position">
+            I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <p><br><a class="intro-connect">LET’S CONNECT</a><br></p>
+        <ul class="view-social-midea">
+            <li class="social-icon">
+                <a><img src="./assets/images/social-icon/twitter-icon.png" alt="twitter"></a>
+            </li>
+            <li class="social-icon">
+                <a><img src="./assets/images/social-icon/linkedin icon.png" alt="linkedin"></a>
+            </li>
+            <li class="social-icon">
+                <a><img src="./assets/images/social-icon/m-icon.png" alt="m"></a>
+            </li>
+            <li class="social-icon">
+                <a><img src="./assets/images/social-icon/github-icon.png" alt="github"></a>
+            </li>
+            <li class="social-icon">
+                <a><img src="./assets/images/social-icon/clip-icon.png" alt="fingers"></a>
+            </li>
+        </ul>
+            </span>
+        </section>
+
+         <section class="grid-item grid-item-3 work-section">
+            <div id="work-container">
```

Comment:
- [ ] KIndly ensure you add the hover interactions on the buttons as shown on the figma design which helps to increase user experience 👍🏽 

![image](https://user-images.githubusercontent.com/63473584/219848498-90424fca-a439-4424-a1e3-6c5f8fa7e9b3.png)